php/WP_CLI/Bootstrap/IncludeBundledAutoloader.php

```diff
@@ -22,25 +22,13 @@
 	 */
 	protected function get_autoloader_paths() {
 		$autoloader_paths = array(
-			// Part of a larger project / installed via Composer (preferred).
-			WP_CLI_ROOT . '/../../../vendor/autoload_commands.php',
-			// Top-level project / installed as Git clone.
-			WP_CLI_ROOT . '/vendor/autoload_commands.php',
+			WP_CLI_VENDOR_DIR . '/autoload_commands.php',
```

Comment:
Hah. I had meant to do this abstraction when `WP_CLI_VENDOR_DIR` originally landed.